.github/workflows/ci2.yml

```diff
@@ -90,48 +58,83 @@ jobs:
 
   Build:
     name: Build DAOS
-    needs: Prepare
     runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
       matrix:
-        # Run this on distros where we don't run the ci1 Workflow.
-        distro: [ubuntu.20.04, centos.8, leap.15]
+        distro: [ubuntu.20.04, centos.8, leap.15, fedora, rocky, alma]
         compiler: [clang, gcc]
+        include:
+          - distro: fedora
+            base: centos.8
+            with: fedora:34
+            java: NO
+          - distro: rocky
```

Comment:
given that RedHat has just released 9.0-beta, should we be prepared for that and rename rocky to rocky.8 and alma to alma.8 ?

Reply:
We should be prepared to update all the upstream versions we're tracking, ideally that should be in a self-contained PR however, and having this code tidy and consistent would help with that.